rclpy/src/rclpy/events_executor/events_executor.cpp

```diff
@@ -75,10 +75,16 @@ pybind11::object EventsExecutor::create_task(
   // manual refcounting on it instead.
   py::handle cb_task_handle = task;
   cb_task_handle.inc_ref();
-  events_queue_.Enqueue(std::bind(&EventsExecutor::IterateTask, this, cb_task_handle));
+  call_task_in_next_spin(task);
   return task;
 }
 
+void EventsExecutor::call_task_in_next_spin(pybind11::handle task)
+{
+  events_queue_.Enqueue(std::bind(&EventsExecutor::IterateTask, this, task));
+}
+
+
```

Comment:
Super tiny nit: unnecessary whitespace

Reply:
Fixed. IMO this should be caught by the formatter but ament_uncrustify is very permissive.